### PR TITLE
Reputation.php warning

### LINKED ISSUE #4758: Reputation.php warning
https://community.mybb.com/thread-239243.html

### PR DESCRIPTION
**Pull request by Eldenroot**
Resolves #4758

### CODE REVIEW

File: reputation.php [hunk 1]
```diff
@@ -166,6 +166,10 @@
 	{
 		$query = $db->simple_select("reputation", "*", "adduid='".$mybb->user['uid']."' AND uid='{$uid}' AND pid='0'");
 		$existing_reputation = $db->fetch_array($query);
+		if(!empty($existing_reputation['rid']))
+		{
+			$rid = $existing_reputation['rid'];
+		}
 		$rid = $existing_reputation['rid'];
```
Comment (Matslom) [inline reputation.php]: You have omitted to remove this line.

Comment (Stefan-MyBB) [reply reputation.php]: Also `if($existing_reputation)` would be shorter and do the same.

Comment (Eldenroot) [reply reputation.php]: Thank you!

**Comment by dvz** [comment]
Should this still be a draft?

**Pull request merged**